src/allowed_args.cpp

```diff
@@ -310,10 +310,15 @@ static void addGeneralOptions(AllowedArgs &allowedArgs, HelpMessageMode mode)
                     DEFAULT_PERSIST_MEMPOOL))
         .addArg("prune=<n>", requiredInt,
             strprintf(_("Reduce storage requirements by pruning (deleting) old blocks. This mode is incompatible with "
-                        "-txindex and -rescan. "
+                        "-txindex and -rescan. Incompatible with -prunewithmask"
                         "Warning: Reverting this setting requires re-downloading the entire blockchain. "
```

Comment:
Nit:  Should it maybe better to say "This mode is incompatible with -txindex, -rescan and -prunewithmask"  .

Reply:
fixed